app/views/catalog/_show_tabs.html.erb

```diff
@@ -1,26 +1,26 @@
 <ul class="lux-tabs-container" role="tablist" aria-label="Collection Notes">
   <li id="summary-tab" class="lux-tabs-item" role='tab'>
-    <a class='nav-link' href='#summary'>
-      Collection Overview
+    <a class='nav-link' href='<%= solr_document_path(document.http_safe_ead) %>'>
```

Comment:
What's `http_safe_ead`? Did `document.id` not work?

Reply:
I needed the Collection URL, and that was the way I found I could get it. The Document.id is the component URL if you're on a component.